### PR TITLE
Fix check_tasks_not_updated() not working

### PR DESCRIPTION
**Pull request by loren**

### CODE REVIEW

File: testing/sdk_tasks.py [hunk 1]
```diff
@@ -66,25 +67,12 @@ def fn():
...
-    try:
-        sdk_spin.time_wait_noisy(lambda: fn(), timeout_seconds=60)
-    except shakedown.TimeoutExpired:
-        sdk_utils.out('Timeout reached as expected')
+    sdk_plan.get_deployment_plan(service_name)
+    task_ids = get_task_ids(service_name, prefix)
+    msg = 'Checking tasks starting with "{}" have not been updated:\n- Old tasks: {}\n- Current tasks: {}'.format(
+        prefix, old_task_ids, task_ids)
+    sdk_utils.out(msg)
+    assert set(old_task_ids).issubset(set(task_ids)), "Tasks got updated"
```
Comment (nickbp) [inline testing/sdk_tasks.py]: Will this assert log the content of the two lists? Probably nice to have

File: testing/sdk_tasks.py [hunk 2]
```diff
@@ -66,25 +67,12 @@ def fn():
...
-        for task_id in task_ids:
-            if task_id not in old_task_ids:
-                return False
-        return True
-
-    try:
-        sdk_spin.time_wait_noisy(lambda: fn(), timeout_seconds=60)
-    except shakedown.TimeoutExpired:
-        sdk_utils.out('Timeout reached as expected')
+    sdk_plan.get_deployment_plan(service_name)
```
Comment (mbdcos) [inline testing/sdk_tasks.py]: Does this mean wait till plan completes?

**Comment by loren** [comment]
The line right before the assert logs the contents of the two lists so it seemed redundant to write out the same data again. I'm happy to do so though if you like.

**Comment by loren** [comment]
retest this please

**Comment by nickbp** [comment]
Oh I didn't notice that. Yeah that's fine

**Comment by loren** [comment]
Thank you for reviewing this, @nickbp and @mbdcos.

**Pull request merged**